libcxx/test/std/library/description/conventions/customization.point.object/cpo.compile.pass.cpp

```diff
@@ -81,25 +83,48 @@ static_assert(test(std::ranges::ssize, a));
 // views::empty<T> is not a CPO
 static_assert(test(std::views::iota, 1));
 static_assert(test(std::views::iota, 1, 10));
-//static_assert(test(std::views::istream<int>, 1);
+static_assert(test(std::views::istream<int>, stream));
```

Comment:
There is a real issue with generic-no-localization. We don't define, but merely forward-declare stream types, when `_LIBCPP_HAS_LOCALIZATION` is not defined. However, we unconditionally provide `istream_view`.